### DISCUSSION TITLE
win11,22H2 13代处理器 无温度显示，强迫症要疯了

### DISCUSSION THREAD

**Discussion by Hemaizi** [zhongyang219/TrafficMonitor#1459] [General]
win11,22H2 13代处理器 无温度显示，强迫症要疯了，使用的是V1.84.1_x64

**Comment by Hemaizi** [comment]
以下载LibreHardwareMonitor软件，并能成功运行，可以看到CPU温度！但是在咱的软件上还是看不到

**Comment by Hemaizi** [comment]
![image](https://user-images.githubusercontent.com/119333535/204305001-ebaaea88-cb4a-4a47-932d-54b3f18e1d1e.png)

**Comment by ElmoRon9** [comment]
我的也是win11 13代U  看不到频率和温度

**Comment by qinghuaci1192** [comment]
把下载的LibreHardwareMonitor里的LibreHardwareMonitorLib.dll复制到TrafficMonitor文件夹里覆盖就好了

**Reply by Mengchuxia** [reply]
可行  刚实验好   以为没人说  还上来看看哈哈哈

**Comment by Ychan98** [comment]
上面提供的方法有用！另外问一句监测温度选哪个比较有代表性啊

**Comment by etls5228702** [comment]
上面提供的方法有用！ 替换过去就有cpu温度了
